src/html.pre.js

```diff
@@ -55,9 +56,29 @@ function pre(context) {
   }
 
   // ensure content.data is present
-  if (!context.content.data) {
-    context.content.data = {};
-  }
+  content.data = content.data || {};
+
+  // extract metadata
+  const { meta = {} } = content;
+  // description: text from paragraphs with 10 or more words
+  const desc = $sections
+    .filter('.default')
+    .find('p')
+    .filter(function minLength() {
+      return $(this).text().split(' ').length >= 10;
+    })
+    .map(function padWithSpaces(i) {
+      if (i > 0) $(this).text(` ${$(this).text()}`);
```

Comment:
missing `{}`

(don't we have eslint enabled?)

Reply:
> (don't we have eslint enabled?)

We do. Weird, it doeesn't complain in VS Code...

Reply:
> btw: not very efficient: you use $(this).text() several times....

Yes,  but once as a setter, once as a getter.